3100. Water Bottles II.cpp is added

Anujmishra2005 opened the pull request:
3100. Water Bottles II.cpp is added



@Anujmishra2005 I mean ..... Add those approach intution in here .... 

Not on the code file

Anujmishra2005 commented:
Added My Intuition and Approach in it also

SjxSubham commented:
@Anujmishra2005 after making those changes 

### Star the repo ⭐
## Happy HAcktober... keep contributing...

Anujmishra2005 commented:
Hi , i have also modified Water Bottles II.cpp in the pull request #45 in which i have written only code in the code file and added My Approach and Intuition and Complexity in the Description of the pull request
